piet-test/src/picture_5.rs

```diff
@@ -21,5 +21,19 @@ pub fn draw(rc: &mut impl RenderContext) -> Result<(), Error> {
     // underline text
     rc.stroke(Line::new((100.0, 52.0), (100.0 + width, 52.0)), &brush, 1.0);
 
+    // get hit test text position and draw a curos
+    // this is for position three, not trailing
+    let hit_test_text_position = layout.hit_test_text_position(3, true);
+
+    if let Some(http) = hit_test_text_position {
+        // TODO should the points be f64? directwrite only returns f32
+        let cursor_x = http.point.x as f64;
+        println!("cursor_x: {}", cursor_x);
```

Comment:
Feels like maybe this should be removed, though I see why you have it during development.